test/test_watch_schedule_builder/test_watch_schedule_builder.py

```diff
@@ -0,0 +1,116 @@
+from ketchup import watch_schedule_builder as wsb
+import datetime as dt
+
+
+def test_calculate_wsb_valid_response():
+    end_date = dt.datetime(2024, 11, 11)
+    start_date = dt.datetime(2024, 11, 1)
+    response = wsb.calculate_watch_schedule(50, end_date, start_date)
+
+    assert response["base_eps_per_day"] == 5
+    assert response["days"] == 10
+    assert response["actual_watch_days"] == 10
+    assert response["is_possible"] is True
+    assert response["realistic_end_date"] is None
+    assert response["extra_episodes"] is None
+
+
+def test_calculate_wsb_valid_response_uneven_days():
+    end_date = dt.datetime(2024, 11, 11)
+    start_date = dt.datetime(2024, 11, 1)
+    response = wsb.calculate_watch_schedule(51, end_date, start_date)
+
+    assert response["base_eps_per_day"] == 6
+    assert response["days"] == 10
+    assert response["actual_watch_days"] == 8.5
+    assert response["is_possible"] is True
+    assert response["realistic_end_date"] is None
+    assert response["extra_episodes"] is None
```

Comment:
Ah this is interesting. Looks like we will need to introduce the extra episodes logic into the easy flow too. I expected this to be "1" extra episodes but I realize it only does that for the limit paths. We can add a ticket to fix that in a different PR